index.html

```diff
@@ -48,6 +48,16 @@
     />
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <style></style>
+
+    <!-- Global site tag (gtag.js) - Google Analytics -->
+    <script async src="https://www.googletagmanager.com/gtag/js?id=UA-117977508-2"></script>
```

Comment:
Do we want a way to turn this off when running the app locally or in a PR branch deployed to now?

Reply:
Or perhaps it's easy enough to filter that traffic out in GA somehow?

Reply:
The latter! It's straight-forward to exclude localhost traffic in the GA Admin UI, e.g. https://briefmetrics.com/articles/remove-localhost-from-referrers.